crates/collab/migrations/20241023201725_remove_dev_servers.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE dev_servers;
+DROP TABLE dev_server_projects;
+
+ALTER TABLE projects DROP COLUMN dev_server_project_id;
```

Comment:
I think we'd need to drop this column before we drop the table that it has a foreign key to.

Reply:
Now you point it out, I'd agree; but it ran fine locally *hmm*. Maybe it's happy for the schema to be invalid within a transaction?

Either way, I'll put it first